package.json

```diff
@@ -143,6 +143,7 @@
     "eslint-import-resolver-typescript": "^3.5.0",
     "eslint-import-resolver-webpack": "^0.13.0",
     "eslint-nibble": "^6.1.0",
+    "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-import": "^2.24.2",
```

Comment:
Why is this new package required?

Reply:
Euh, it's not required as I found out just now. If I remember correctly it got installed after I clicked fix all import problems. I wasn't sure if they were really needed since we don't run the end2end-tests now but I figured better safe then sorry. (and Psycharm did flag it as a problem. I've removed them. Tests work and website is also running just fine.